modules/gecco.nf

```diff
@@ -0,0 +1,43 @@
+process GECCO_RUN {
+
+    tag "${cluster_name}"
+    
+    publishDir(
+        path: "${params.outdir}",
+        saveAs: {
+            filename -> {
+                String cluster_prefix = cluster_name.substring(0, cluster_name.length() - 2);
+                return "species_catalogue/${cluster_prefix}/${cluster_name}/genome/${cluster_name}_gecco.gff"
+            }
+        },
+        mode: 'copy',
+        failOnError: true
+    )
+
+    container 'https://depot.galaxyproject.org/singularity/gecco:0.9.8--pyhdfd78af_0'
```

Comment:
```suggestion
    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
        'https://depot.galaxyproject.org/singularity/gecco:0.9.8--pyhdfd78af_0':
        'biocontainers/gecco:0.9.8--pyhdfd78af_0' }"
```
we need to fix the docker support, so let's keep the docker images here to make that easier